marge/batch_job.py

```diff
@@ -108,6 +108,19 @@ def ensure_mr_not_changed(self, merge_request):
             if getattr(changed_mr, attr) != getattr(merge_request, attr):
                 raise CannotMerge(error_message.format(attr.replace('_', ' ')))
 
+    def merge_batch(self, target_branch, source_branch, no_ff=False):
+        if no_ff:
+            return self._repo.merge(
+                    target_branch,
+                    source_branch,
+                    '--no-ff',
+                )
```

Comment:
Formatting is off a bit here. should look like the block below, with closing `)` aligned with `return`